src/Features/Core/Portable/IntroduceVariable/AbstractIntroduceParameterService.cs

```diff
@@ -69,6 +69,13 @@ public sealed override async Task ComputeRefactoringsAsync(CodeRefactoringContex
                 return;
             }
 
+            // Need to special case for highlighting of method types because they are also "contained" within a method,
+            // but it does not make sense to introduce a parameter in that case.
+            if (syntaxFacts.IsInNamespaceOrTypeContext(expression))
+            {
+                return;
+            }
```

Comment:
```csharp
void M()
{
    var x = typeof(string);
}
```

Is `typeof(string)` node considered a type context? I expect add parameter to work here and transforms the code into the following (or similar):

```csharp
void M(System.Type param)
{
    var x = param;
}
```

Reply:
>Is typeof(string) node considered a type context?

The `string` part would be a type context.  the `typeof(string)` itself would not be.  But @akhera99 can you add tests to verify that?

Reply:
Correct, the entire expression is not a Type, so it is fine. Added a test for it.